apis/kueue/v1alpha2/localqueue_types.go

```diff
@@ -35,6 +35,11 @@ type LocalQueueStatus struct {
 	// localQueue not yet admitted to a ClusterQueue.
```

Comment:
not sure why it says admitted.

It should say:

```
pendingWorkloads is the number of Workloads in the LocalQueue not yet admitted to a ClusterQueue
```

Reply:
please fix the comment for `pendingWorkloads`

Reply:
Sorry about that.  Missed this earlier!